packages/app/src/feature-flags/features/index.ts

```diff
@@ -7,88 +7,169 @@ import { EnvironmentConfig } from "App/feature-flags/types"
+  [Feature.LogsScrubbingEnabled]: {
     [Environment.Development]: false,
     [Environment.Production]: true,
-    [Environment.TestProduction]: true,
     [Environment.AlphaProduction]: true,
-    [Environment.TestAlphaProduction]: true,
   },
-  [Feature.DeveloperModeHidden]: {
-    [Environment.Development]: false,
-    [Environment.Production]: true,
-    [Environment.TestProduction]: true,
+  [Feature.DeveloperModeEnabled]: {
+    [Environment.Development]: true,
+    [Environment.Production]: false,
+    [Environment.AlphaProduction]: true,
+  },
+  [Feature.FilesManagerEnabled]: {
+    [Environment.Development]: true,
+    [Environment.Production]: false,
     [Environment.AlphaProduction]: false,
-    [Environment.TestAlphaProduction]: false,
   },
-  [Feature.DisabledOnProduction]: {
-    [Environment.Development]: false,
-    [Environment.Production]: true,
-    [Environment.TestProduction]: true,
+  [Feature.MessagesTemplatesTabEnabled]: {
+    [Environment.Development]: true,
+    [Environment.Production]: false,
     [Environment.AlphaProduction]: false,
-    [Environment.TestAlphaProduction]: false,
   },
-  [Feature.McPrereleaseAvailable]: {
+  [Feature.MessagesTabEnabled]: {
     [Environment.Development]: true,
     [Environment.Production]: false,
-    [Environment.TestProduction]: true,
-    [Environment.AlphaProduction]: true,
-    [Environment.TestAlphaProduction]: true,
+    [Environment.AlphaProduction]: false,
   },
-  [Feature.MessagesHidden]: {
-    [Environment.Development]: false,
+  [Feature.MuditaCenterPrereleaseEnabled]: {
+    [Environment.Development]: true,
     [Environment.Production]: false,
-    [Environment.TestProduction]: false,
     [Environment.AlphaProduction]: false,
-    [Environment.TestAlphaProduction]: false,
   },
-  [Feature.DevelopOnly]: {
+  [Feature.MessagesThreadAttachContactEnabled]: {
     [Environment.Development]: true,
     [Environment.Production]: false,
-    [Environment.TestProduction]: false,
     [Environment.AlphaProduction]: false,
-    [Environment.TestAlphaProduction]: false,
   },
-  [Feature.ProductionAndAlpha]: {
-    [Environment.Development]: false,
-    [Environment.Production]: true,
-    [Environment.TestProduction]: true,
-    [Environment.AlphaProduction]: true,
-    [Environment.TestAlphaProduction]: true,
+  [Feature.MessagesThreadAttachTemplateEnabled]: {
+    [Environment.Development]: true,
+    [Environment.Production]: false,
+    [Environment.AlphaProduction]: false,
   },
-  [Feature.MessagesSearch]: {
+  [Feature.MessagesThreadCallsEnabled]: {
     [Environment.Development]: true,
     [Environment.Production]: false,
-    [Environment.TestProduction]: false,
     [Environment.AlphaProduction]: false,
-    [Environment.TestAlphaProduction]: false,
   },
-  [Feature.ProductionReleaseOnly]: {
-    [Environment.Development]: false,
-    [Environment.Production]: true,
-    [Environment.TestProduction]: false,
+  [Feature.MessagesThreadMarkAsReadEnabled]: {
```

Comment:
[Mudita Center v1.4.0-rc.1](https://github.com/mudita/mudita-center/releases/tag/1.4.0-rc.1) has read/unread feature from thread dropdown, so here it can be confusing. I suggest changing name.
```suggestion
  [Feature.MessagesThreadDetailsMarkAsReadEnabled]: {
```

Reply:
https://github.com/mudita/mudita-center/pull/782/commits/f5370f0ca35a5ec0db57208152de995e3a9249d8